plugin_blacklist:
- '3dr_radio'
- 'image_pub'

# vim: set noet fenc=utf-8 ff=unix ft=yaml :
